### PR TITLE
_WD_FrameEnter - fix for {"id":null}

### PR DESCRIPTION
**Pull request by mlipok**
Fixed NULL usage and added Remarks

### CODE REVIEW

**Review by Danp2 — COMMENTED** [review]
Thanks for catching my error with null. Regarding your "remarks" section, I'm unsure if it helps or just adds to the confusion.

File: wd_helper.au3 [hunk 1]
```diff
@@ -509,8 +509,8 @@ EndFunc   ;==>_WD_IsWindowTop
 ;                  @ERROR       - $_WD_ERROR_Success
 ;                  				- $_WD_ERROR_Exception
 ; Author ........: Decibel
-; Modified ......: 2018-04-27
-; Remarks .......:
+; Modified ......: mLipok
+; Remarks .......: For nested frames use this function as many time as many nesting level is in analyzed website
```
Comment (mlipok) [inline wd_helper.au3:513]: `You can drill-down into nested frames by calling this function repeatedly with the correct parameters`
will be better

Comment (Danp2) [reply wd_helper.au3:513]: @mlipok Pretty sure that you just submit an additional commit to the same branch (https://github.com/mlipok/WebDriver/tree/patch-4)

Comment (mlipok) [reply wd_helper.au3]: I hope I do exactly what you mean, if not please provide me a hint.

**Review by Danp2 — APPROVED** [review]

**Pull request merged**